test/tools/d_do_test.d

```diff
@@ -612,9 +620,33 @@ bool compareOutput(string output, string refoutput, const ref EnvData envData)
             if (!parts || !exists(parts[0]))
                 return false;
 
-            refoutput = refparts[2];
             output = parts[1];
```

Comment:
A continue here would eliminate the need for the `else` below

Reply:
Done